test/unit/units/prepare_test.py

```diff
@@ -100,6 +110,30 @@ def test_main(
                     'update-ca-certificates'
                 ]
             ),
+            call(
+                [
+                    'ip', 'a'
+                ],
+                raise_on_error=False
+            ),
+            call(
+                [
+                    'ip', 'r'
+                ],
+                raise_on_error=False
+            ),
+            call(
+                [
+                    'cat', '/etc/resolv.conf'
+                ],
+                raise_on_error=False
+            ),
+            call(
+                [
+                    'cat', '/proc/net/bonding/bond*'
+                ],
+                raise_on_error=False
+            ),
```

Comment:
I think it can be written in one line

```python
call(
    ['cat', '/proc/net/bonding/bond*'], raise_on_error=False
)
```

The 80 characters per line limit is not violated iirc

Reply:
I tried to follow the same pattern that it was in the test already, that doesn't violate the limit either.
I will update it.

Reply:
right, my bad... happy if you fix my stuff too :)